src/shared/extensionUtilities.ts

```diff
@@ -38,8 +41,10 @@ export enum IDE {
     unknown,
 }
 
+let computeRegion: string | undefined = NOT_INITIALIZED
```

Comment:
could use `undefined` to mean "not initialized", and empty string to mean "unknown / N/A".

Reply:
Could do that. FWIW `unknown` currently represents a Cloud9 instance that can't contact the metadata service, and `undefined` represents a non-Cloud9 IDE.